package.json

```diff
@@ -5,8 +5,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test1": "mocha",
-    "test": "c8 --all mocha"
+    "test": "c8 --all mocha",
+    "testNumberGame": "c8 --include numberGame.js mocha test/numberGame.test.js"
```

Comment:
(no idea myself) do you need the include? would it work without it?

Reply:
fixed